updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,171 @@
+:_content-type: ASSEMBLY
+[id="mirroring-ocp-image-repository"]
+= Mirroring the OpenShift Container Platform image repository
+include::_attributes/common-attributes.adoc[]
+:context: mirroring-ocp-image-repository
+
+toc::[]
+
+You must mirror container images onto a mirror registry before you can update a cluster in a disconnected environment. You can also use this procedure in connected environments to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[NOTE]
+====
+This registry must be running at all times as long as the cluster is running.
+====
+
+There are two supported methods for mirroring images onto a mirror registry:
+
+* Using the oc-mirror OpenShift CLI (`oc`) plugin
+
+* Using the `oc adm release mirror` command
+
+Compared to using `oc adm release mirror`, the oc-mirror plugin has the following advantages:
+
+* It is simpler to use.
```

Comment:
I'd be careful about a blanket statement like this. It also doesn't really look simpler in this assembly when there are a ton of sections for oc-mirror, but only one for oc adm release :wink:

Reply:
That's a good point. I'll ask a developer to help me specify the ways in which it is simpler to use, and if the answer is too tricky to fit into a bullet, I'll just delete this point entirely.